docs/build/configure/triggers.md

```diff
@@ -193,23 +222,65 @@ If you prefer to configure your trigger with raw JSON instead of the config buil
        "Location-ID": request.headers['location-id'] if 'location-id' in request.headers else 'no value',
        "Part-ID": request.headers['part-id'] if 'part-id' in request.headers else 'no value',
        "Robot-ID": request.headers['robot-id'] if 'robot-id' in request.headers else 'no value',
-       "Machine-Name": request.headers['machine-name'] if 'machine-name' in request.headers else 'no value',
-       "Location-Name": request.headers['location-name'] if 'location-name' in request.headers else 'no value',
-       "Organization-Name": request.headers['organization-name'] if 'organization-name' in request.headers else 'no value'
+       "Component-Type": request.headers['component-type'] if 'component-type' in request.headers else 'no value',
+       "Component-Name": request.headers['component-name'] if 'component-name' in request.headers else 'no value',
+       "Method-Name": request.headers['method-name'] if 'method-name' in request.headers else 'no value',
+       "Min-Time-Received": request.headers['min-time-received'] if 'min-time-received' in request.headers else 'no value',
+       "Max-Time-Received": request.headers['max-time-received'] if 'max-time-received' in request.headers else 'no value',
+
+      "Data-Type": request.args['data_type'] if 'data_type' in request.args else 'no value'
      }
 
-     slack_url = "<paste in your own Slack URL>"
+     trigger_url = "<paste in your own trigger URL>"
      headers = {}
 
-     response = requests.post(slack_url, json=payload, headers=headers)
+     response = requests.post(trigger_url, json=payload, headers=headers)
 
      request_json = request.get_json(silent=True)
      request_args = request.args
 
-     return 'Sent request to {}'.format(slack_url)
+     return 'Sent request to {}'.format(trigger_url)
```

Comment:
Because you're tailoring the example to be "sends a message" --- it needs to be clear that this URL represents a recipient of a message. I see two options here - either (1) keep the example as a slack message and use the above suggestion to be clear to the reader that this example is for a slack message specifically, or (2) make the example a _**generic**_ message and get rid of the concept of a trigger URL and remove the post request (because those are specific to a kind of action that can be taken in the lambda)